stackstorm/Dockerfile

```diff
@@ -8,7 +8,7 @@ LABEL com.stackstorm.version="${ST2_VERSION}"
 
 ADD ./sources.list /etc/apt/ 
 
-RUN apt-get -y update && apt-get -y install adduser procps python2.7 libffi5 libyaml-0-2 git libpcre3 libpython2.7 libxml2 python-dev libssl-dev libffi-dev sudo
+RUN apt-get -y update && apt-get -y install adduser procps python2.7 libffi5 libyaml-0-2 git libpcre3 libpython2.7 libxml2 python-dev libssl-dev libffi-dev sudo netbase
```

Comment:
This change is not needed.
Please remove it to avoid  changelog/confusion.

Reply:
Thanks for the heads up - will remove it.